src/backend/lib/prisma.ts

```diff
@@ -0,0 +1,9 @@
+import { PrismaClient } from '@prisma/client';
+
+const globPrisma = global as unknown as { prisma: PrismaClient };
```

Comment:
wtf is as unknown as

Reply:
lol, be nice people might see this.

In order to give globPrisma a type I have to first give it an unknown type.

that's because it's already typed - global is the node native global object.

this file is actually from the Prisma docs:
https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#prismaclient-in-long-running-applications